ixmp/core.py

```diff
@@ -55,6 +55,8 @@ class Platform:
     _backend_direct = [
         'open_db',
         'close_db',
+        'get_doc',
```

Comment:
I'm confused, is there now a top-level function to get or set docstrings?

Reply:
Ok, thanks for the clarification - how is this rendered in the docs? Is there an example where I can see this already?

Reply:
Good question! I think it is not available on [Platform class documentation](https://message.iiasa.ac.at/projects/ixmp/en/stable/api-python.html#platform).

@khaeru what do you think, can we adjust `__doc__` attribute to incorporate those backend methods documentation into Platform docs?